src/reflect/scala/reflect/internal/Depth.scala

```diff
@@ -31,6 +31,7 @@ final class Depth private (val depth: Int) extends AnyVal with Ordered[Depth] {
   override def toString = s"Depth($depth)"
 }
 
+@FunctionalInterface
```

Comment:
Do we give effect to this?

Reply:
yes, for SAM consideration, because WWJD.